x-pack/plugins/maps/public/connected_components/edit_layer_panel/layer_settings/layer_settings.tsx

```diff
@@ -50,6 +52,13 @@ export function LayerSettings(props: Props) {
     props.updateLabel(layerId, label);
   };
 
+  const onColorChange = (color: string) => {
+    // TODO default operation does not change when selecting new basemap style
+    // not sure how to resolve this?
```

Comment:
Each tile style has a preferred filter operation. But it currently does not select the preferred operation when changing the style in the basemap settings. I'm stuck trying to resolve this and I think it is a big enough concern to block merging until it is resolved.